src/app/components/TextToCopy.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { MdContentCopy } from "react-icons/md";
+import Tippy from "@tippyjs/react";
+import "tippy.js/dist/tippy.css";
+import "tippy.js/dist/svg-arrow.css";
+import { useTranslations } from "../hooks";
+
+interface TextToCopyProps {
+  targetUrl: string;
+}
+
+export function TextToCopy({ targetUrl }: TextToCopyProps) {
```

Comment:
Sorry I think this was a misunderstanding, I meat to rename `targetUrl` -> `textToCopy`. The component being named "CopyIcon" was fine.

Please change to:

```suggestion
interface CopyIconProps {
  textToCopy: string;
}

export function CopyIcon({ textToCopy }: CopyIconProps) {
```